### PR TITLE
Correct base_url

### PR DESCRIPTION
**Pull request by ritwik12**

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -47,7 +47,7 @@ url:
 # (http://yourusername.github.io/repository-name)
 # and NOT your User repository (http://yourusername.github.io)
 # then add in the baseurl here, like this: "/repository-name"
-baseurl: "http://riya-17.github.io/My-Blog"
+baseurl: "/My-Blog"
```
Comment (riya-17) [inline _config.yml:50]: Are you sure about this?

Comment (ritwik12) [reply _config.yml:50]: yes, mine is this only

Comment (ritwik12) [reply _config.yml:50]: and read above comments too @riya-17

**Comment by ritwik12** [comment]
@riya-17 If you go to your blog URL https://riya-17.github.io/My-Blog/ in Chrome and right click and go to inspect.
Then you can see two errors as shown below in console. This shows why this PR is needed.

```
riya-17.github.io/:23 Mixed Content: The page at 'https://riya-17.github.io/My-Blog/' was loaded over HTTPS, but requested an insecure stylesheet 'http://riya-17.github.io/My-Blog/style.css'. This request has been blocked; the content must be served over HTTPS.
riya-17.github.io/:1 Mixed Content: The page at 'https://riya-17.github.io/My-Blog/' was loaded over HTTPS, but requested an insecure stylesheet 'http://riya-17.github.io/My-Blog/style.css'. This request has been blocked; the content must be served over HTTPS.
```

File: _config.yml (hunk 1 above)
Comment (riya-17) [reply _config.yml:50]: ok

**Comment by riya-17** [comment]
Ok, Thanks for your PR

**Pull request merged**